### DISCUSSION TITLE
Installation on macOS

### DISCUSSION THREAD

**Discussion by Brubeck48** [rtl-airband/RTLSDR-Airband#299] [Q&A] [answered]
Could you provide please instruction instruction for macOS?

I tried, but I got only errors with dependancies. Unfortunately I can't find all of them in homebrew for example.

**Comment by szpajder** [comment, marked as answer]
This should suffice:

brew install lame libshout libconfig fftw librtlsdr soapysdr pulseaudio

**Reply by Brubeck48** [reply]
Thank you! It's working on my MacOS Monterey. I think, you can add this instruction to the Readme.
